terraform/aptos-node-testnet/addons.tf

```diff
@@ -178,6 +178,15 @@ resource "helm_release" "testnet-addons" {
         acm_certificate          = length(aws_acm_certificate.ingress) > 0 ? aws_acm_certificate.ingress[0].arn : null
         loadBalancerSourceRanges = var.client_sources_ipv4
       }
+      load_test = {
+        fullnodeGroups = var.aptos_node_helm_values.fullnode.groups
```

Comment:
afaik this might result in error since `aptos_node_helm_values` has no explicit type and might not have a  `fullnode.groups`